assets/tools/rad-validate

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+echo "* locating oc command"
+if ocloc=$(which oc) &> /dev/null; then
+    echo "  success: $ocloc"
+else
+    echo "  error: the oc command must be in your path"
+    exit 1
+fi
+
+echo "* checking OpenShift status"
+if oc status &> /dev/null; then
+    echo "  success: status acquired"
+else
+    echo "  error: unable to determine OpenShift status, perhaps you are not logged in?"
+    exit 1
+fi
+
+echo "* scanning for radanalytics.io objects"
+rad_valid=yes
+missing_objects=""
+
+check_component()
+{
+    if oc get "$1" "$2" &> /dev/null; then
+        if [[ "$rad_valid" == "yes" ]]; then
+            rad_valid=yes
+        fi
```

Comment:
hmm, these 3 lines are not necessary, are they?

Reply:
mostly leftover from testing, i can remove them